eth/tracers/api.go

```diff
@@ -438,6 +439,20 @@ func (api *API) TraceBlockByNumber(ctx context.Context, number rpc.BlockNumber,
 	if err != nil {
 		return nil, err
 	}
+
+	if api.backend.ChainConfig().IsOptimismPreBedrock(block.Number()) {
```

Comment:
```suggestion
	if api.backend.ChainConfig().IsPreKromaMPT(block.Number()) {
```
Shouldn't it be IsPreKromaMPT?